platforms/KallistiOS.cmake

```diff
@@ -18,6 +18,9 @@ option(RSDK_DEBUG "Enable debugging" ON)
 target_compile_definitions(RetroEngine PUBLIC RSDK_DEBUG=$<BOOL:${RSDK_DEBUG}>)
 target_compile_definitions(${GAME_NAME} PUBLIC RSDK_DEBUG=$<BOOL:${RSDK_DEBUG}>)
 
+option(KOS_USER_DIR "Root directory fo the KOS VFS which get set as the RSDK User directory." "/pc/")
```

Comment:
This will default to `/pc/` if the flag is omitted, you probably meant `/cd/`.
(Also typo: fo → for)

Reply:
Goddamnit. Correct.